lib/ansiblelint/rules/TrailingWhitespaceRule.py

```diff
@@ -29,6 +33,6 @@ class TrailingWhitespaceRule(AnsibleLintRule):
     tags = ['formatting', 'ANSIBLE0002']
     version_added = 'historic'
 
-    def match(self, file, line):
+    def match(self, file: Optional["TargetFile"] = None, line: str = "") -> Union[bool, str]:
```

Comment:
I think if the implementation does not actually return strings, it should be just
```suggestion
    def match(self, file: Optional["TargetFile"] = None, line: str = "") -> bool:
```